style/main.css

```diff
@@ -194,6 +194,30 @@ form[role="dialog"][data-type="confirm"].lcdui-alert section {
   left: calc(50% - 17px);
 }
 
+.background-screen {
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  bottom: auto;
+  background-color: white;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-image: url("../img/icon-128.png");
+}
+
+.debug-mode .background-screen {
+  width: 240px;
+  height: 320px;
+}
+
+.autosize .background-screen {
+  display: block;
```

Comment:
`block` should be the default value of *display* for this *div*. Do we really need to specify it here?